app/risk_levels/mcas_risk_level.rb

```diff
@@ -1,18 +0,0 @@
-class McasRiskLevel < Struct.new :student_assessment
```

Comment:
I'm learning that `delegate` is a bad smell so I think it's great that you're getting rid of these mini-classes.

Reply:
Yeah, not a big fan of the Struct.new shortcut. Figured it was just the Rails way. 😄